srcpkgs/lxqt-build-tools-qt5/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'lxqt-build-tools-qt5'
 pkgname=lxqt-build-tools-qt5
-version=0.13.0
+version=2.1.0
```

Comment:
this version is not for qt5: https://github.com/lxqt/lxqt-build-tools/releases/tag/2.0.0